test/e2e/create_test.go

```diff
@@ -245,6 +245,14 @@ func (tc *testContext) provisionBYOHConfigMapWithMachineSet() error {
 	if err != nil {
 		return errors.Wrapf(err, "unable to change default shell of machine %s", machines.Items[0].GetName())
 	}
+	if tc.CloudProvider.GetType() == config.VSpherePlatformType {
+		// By default the Windows Defender antivirus is running on VMs in the test suite. Disable it for a BYOH node to
+		// validate instances are able to be configured into Nodes regardless of the Windows Defender antivirus status.
+		err = tc.disableWindowsDefender(&machines.Items[0])
+		if err != nil {
+			return errors.Wrapf(err, "unable to disable Windows Defender on machine %s", machines.Items[0].GetName())
+		}
+	}
```

Comment:
Do you need a delay or ideally a check if Machine is `running`  here before creating the ConfigMap to avoid potential flakes? I would think not but just checking. My worry is SSH starts too early and we start seeing red herrings in the logs that we will only notice debugging an orthogonal issue.

Reply:
I had considered this and as far as I can see, the wait+check is not needed. After we create the ConfigMap, the operator will try to establish SSH connection to the machine, [retrying for up to 10 mins](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/windows/connectivity.go#L87) which is far greater than the time it takes for a machine to reboot. During all this, the test suite itself is [waiting for the BYOH instance to be configured as a node (for up to 20 mins)](https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/create_test.go#L198-L203) so the risk of flaking is minimal.